docs/core/project-sdk/overview.md

```diff
@@ -16,12 +16,12 @@ The following SDKs are available:
 
 | ID                         | Description                                             | Repo                                   |
 |----------------------------|---------------------------------------------------------|----------------------------------------|
-| `Aspire.AppHost.Sdk`       | The .NET [Aspire SDK](/dotnet/aspire/sdk)               | <https://github.com/dotnet/aspire>     |
 | `Microsoft.NET.Sdk`        | The .NET SDK                                            | <https://github.com/dotnet/sdk>        |
 | `Microsoft.NET.Sdk.Web`    | The .NET [Web SDK](/aspnet/core/razor-pages/web-sdk)    | <https://github.com/dotnet/sdk>        |
 | `Microsoft.NET.Sdk.Razor`  | The .NET [Razor SDK](/aspnet/core/razor-pages/sdk)      | <https://github.com/dotnet/aspnetcore> |
 | `Microsoft.NET.Sdk.BlazorWebAssembly` | The .NET [Blazor WebAssembly SDK](/aspnet/core/blazor#blazor-webassembly) | <https://github.com/dotnet/aspnetcore> |
 | `Microsoft.NET.Sdk.Worker` | The .NET [Worker Service](../extensions/workers.md) SDK |                                        |
+| `Aspire.AppHost.Sdk`       | The .NET Aspire SDK                                     | <https://github.com/dotnet/aspire>     |
```

Comment:
I originally placed this on the first row, sorted alphabetically. Is the logic to move it lower since the others are more common?

Reply:
Yes that's why.